### PR TITLE
Added Heroku button

### PR DESCRIPTION
**Pull request by 4p00rv**
Done! Easy peasy :D

### CODE REVIEW

**Comment by ilyakava** [comment]
Thanks!

Don't forget [environment variables](https://devcenter.heroku.com/articles/app-json-schema#env). Feel free to use defaults from the [README](https://github.com/ilyakava/sup#environment-variables) only `COMPANY_NAME` and `COMPANY_NAME_POSSESSIVE` should be not null, I think...

Is there any way to autofill `PRODUCTION_DOMAIN` and `SMTP_DOMAIN`? Either via the heroic button, or maybe (in a separate PR) via rails conventions in the code?

Also I have a vision of someone clicking the button, and being prompted to enter in environment variables to override the defaults, does Heroku do this at all in some pretty web form?

Lastly, is there support for auto creating the [cron tasks](https://github.com/ilyakava/sup#adding-cron-tasks-heroku)?

**Comment by 4p00rv** [comment]
As it turns out that none of the environment variables are necessary. Also setting one up would be redundant since heroku doesn't allow editing of `env_var`.
My only concerns are setting `PRODUCTION_DOMAIN` and `SMTP_DOMAIN`, which will vary. I couldn't find a way to set that up. Also next thing I found important was `groups`. May be we can create a step by step guide in the app to set all of them up!

**Comment by dblock** [comment]
The ENV vars aren't necessary but I think they are desired to customize things like company name, no?

Also squash your commits please.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.org/ilyakava/sup.svg?branch=generalize)](https://travis-ci.org/ilyakava/sup)
 [![Stories in Ready](https://badge.waffle.io/ilyakava/sup.png?label=ready&title=Ready)](https://waffle.io/ilyakava/sup)
-
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (dblock) [inline README.md:3]: You don't like spaces? Add an empty line after this one back :) Let the code breathe.

**Comment by 4p00rv** [comment]
I have a habit of removing extra lines, adding it back. 
Adding `ENV` vars would be completely redundant in the sense that the company name will vary completely for each deployment. I can add a dummy name if you insist, but would be redundant since the user have to override it.

**Comment by dblock** [comment]
Oh I think Heroku button lets you declare the ENV and then the user fills it when they click, no? I am just thinking out loud.

**Comment by 4p00rv** [comment]
Let me test it out. I will be squashing commits later on.

**Comment by ilyakava** [comment]
When you test it out do you mind taking some screenshots? I too thought that some form on Heroku comes up to fill in the ENV vars...

**Comment by 4p00rv** [comment]
Yes it does come up. I might have missed it.

**Comment by 4p00rv** [comment]
This should do it.
I have added a description accordingly, please tell me if I should change any of those.

**Comment by 4p00rv** [comment]
Hi @ilyakava Can you review this?

**Comment by ilyakava** [comment]
@4p00rv any chance that you can post a video/gif of what the user experience is like?

**Comment by ilyakava** [comment]
Am closing this b/c of inactivity. This would be merge-able for me if there was a companion video/gif showing the user experience start to finish (which could also be placed in the readme)

**Comment by dblock** [comment]
@ilyakava Is that really required? This works just fine.

**Comment by ilyakava** [comment]
@dblock if you've confirmed this works as expected go ahead and merge

**Comment by dblock** [comment]
I haven't ;)

**Pull request closed without merging**